src/pages/home/sidebar/SidebarScreen/index.js

```diff
@@ -0,0 +1,29 @@
+import React, {useRef} from 'react';
+import BaseSidebarScreen from './BaseSidebarScreen';
+
+export default function (props) {
```

Comment:
Name the function

Reply:
I am very embarrassed for the mistake I made. Please accept my apology :( .

Reply:
No worries. Happens with the best of us.